deploy/olm-catalog/host-operator/0.0.1/host-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -2,7 +2,94 @@ apiVersion: operators.coreos.com/v1alpha1
+                      {
+                        "revision": "abcdef",
+                        "type": "ide"
+                      },
+                      {
+                        "revision": "1d2f3q",
+                        "type": "cicd"
+                      },
+                      {
+                        "revision": "a34r57",
+                        "type": "stage"
+                      },
+                      {
+                        "revision": "ra24qw",
+                        "type": "default"
+                      }
+                    ],
+                    "tierName": "basic"
+                  },
+                  "userID": "1a03ecac-7c0b-44fc-b66d-12dd7fb21c40"
+                },
+                "syncIndex": "135540409",
+                "targetCluster": "east-2a"
+              }
+            ],
+            "userID": "1a03ecac-7c0b-44fc-b66d-12dd7fb21c40"
+          }
+        },
+        {
+          "apiVersion": "toolchain.dev.openshift.com/v1alpha1",
+          "kind": "NSTemplateTier",
+          "metadata": {
+            "name": "basic"
+          },
+          "spec": {
+            "namespaces": [
+              {
+                "revision": "abcdef",
+                "template": "{\"apiVersion\":\"v1\",\"kind\":\"Template\",\"metadata\":{\"name\":\"ide-template\"},\"objects\":{\"apiVersion\":\"v1\",\"kind\":\"ProjectRequest\",\"metadata\":{\"name\":\"$USERNAME-ide\"}}}\n",
+                "type": "ide"
+              },
+              {
+                "revision": "1d2f3q",
+                "template": "{\"apiVersion\":\"v1\",\"kind\":\"Template\",\"metadata\":{\"name\":\"cicd-template\"},\"objects\":{\"apiVersion\":\"v1\",\"kind\":\"ProjectRequest\",\"metadata\":{\"name\":\"$USERNAME-cicd\"}}}\n",
+                "type": "cicd"
+              },
+              {
+                "revision": "a34r57",
+                "template": "{\"apiVersion\":\"v1\",\"kind\":\"Template\",\"metadata\":{\"name\":\"stage-template\"},\"objects\":{\"apiVersion\":\"v1\",\"kind\":\"ProjectRequest\",\"metadata\":{\"name\":\"$USERNAME-stage\"}}}\n",
+                "type": "stage"
+              },
+              {
+                "revision": "ra24qw",
+                "template": "{\"apiVersion\":\"v1\",\"kind\":\"Template\",\"metadata\":{\"name\":\"default-template\"},\"objects\":{\"apiVersion\":\"v1\",\"kind\":\"ProjectRequest\",\"metadata\":{\"name\":\"$USERNAME\"}}}\n",
+                "type": "default"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "toolchain.dev.openshift.com/v1alpha1",
+          "kind": "UserSignup",
+          "metadata": {
+            "name": "johnsmith-at-redhat-com"
+          },
+          "spec": {
+            "approved": true,
+            "targetCluster": "east-2a",
+            "userID": "1a03ecac-7c0b-44fc-b66d-12dd7fb21c40",
+            "username": "johnsmith@redhat.com"
+          }
+        }
+      ]
```

Comment:
Did you format it manually? Or is it an output of the generation script?

Reply:
The script generated this.